documentation/src/main/asciidoc/userguide/chapters/fetching/Fetching.adoc

```diff
@@ -204,14 +217,6 @@ include::{sourcedir}/GraphFetchingTest.java[tags=fetching-strategies-dynamic-fet
 ----
 ====
 
-[NOTE]
-====
-When executing a JPQL query, if an EAGER association is omitted, Hibernate will issue a secondary select for every association needed to be fetched eagerly,
-which can lead to N+1 query issues.
-
-For this reason, it's better to use LAZY associations, and only fetch them eagerly on a per-query basis.
-====
-
```

Comment:
The above content is outdated for it only applies when `fetch graph` is not correctly implemented.